src/utils/NetworkUtils.ts

```diff
@@ -123,6 +123,56 @@ export function chainIsL1(chainId: number): boolean {
   return [CHAIN_IDs.MAINNET, CHAIN_IDs.SEPOLIA].includes(chainId);
 }
 
+/**
+ * Determines whether a chain ID runs on an EVM-like execution layer.
+ * @param chainId Chain ID to evaluate.
+ * @returns True if chain corresponding to chainId has an EVM-like execution layer.
+ */
+export function chainIsEvm(chainId: number): boolean {
+  const {
+    ALEPH_ZERO,
+    ARBITRUM,
+    BASE,
+    BLAST,
+    BOBA,
+    INK,
+    LINEA,
+    LISK,
+    MAINNET,
+    MODE,
+    OPTIMISM,
+    POLYGON,
+    REDSTONE,
+    SCROLL,
+    SONEIUM,
+    SUPERSEED,
+    WORLD_CHAIN,
+    ZK_SYNC,
+    ZORA,
+  } = CHAIN_IDs;
+  return [
```

Comment:
This could be shortened by just checking for ChainFamily OP_STACK, ZK_STACK or ORBIT, and then additionall checking for MAINNET, SCROLL or LINEA. That'd be a little bit more maintainable than an explicit list, but I still think we can do better.

Reply:
Fair enough. Alternatively, we can just make a chain family for Solana (and other non evm chains in the future) and check that it isn't in those set. https://github.com/across-protocol/sdk/pull/831/commits/bdaa603fb53c95641090caaf99d9172a03f3c616